mule-packager/src/test/java/org/mule/tools/api/packager/PackagerTestUtils.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.tools.api.packager;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+
+import java.io.IOException;
+import java.nio.file.Path;
+
+/**
+ * This class contains static methods used by all the test classes in the package org.mule.tools.packager
+ */
+public class PackagerTestUtils {
+
+  public static final String CLASSES = "classes";
+
+  public static final String TEST_CLASSES = "test-classes";
+
+  public static final String SRC = "src";
+  public static final String MAIN = "main";
+
+  public static final String MULE = "mule";
+  public static final String POLICY = "policy";
+
+  public static final String TEST = "test";
+  public static final String TEST_MULE = "test-mule";
+  public static final String MUNIT = "munit";
+
+  public static final String REPOSITORY = "repository";
+
+  public static final String MAVEN = "maven";
+  public static final String MULE_SRC = "mule-src";
+  public static final String MULE_ARTIFACT = "mule-artifact";
```

Comment:
this two should probably go (MULE_SRC, MULE_ARTIFACT)

Reply:
Done